lib/screens/home/home.dart

```diff
@@ -1,122 +1,40 @@
 import 'package:dynamic_theme/dynamic_theme.dart';
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart' hide showSearch;
+import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:groovin_material_icons/groovin_material_icons.dart';
 import 'package:pub_client/pub_client.dart';
-import 'package:shared_preferences/shared_preferences.dart';
+import 'package:tavern/screens/bloc.dart';
 import 'package:tavern/screens/search_screen.dart';
 import 'package:tavern/widgets/main_drawer.dart';
 import 'package:tavern/widgets/material_search.dart';
-import 'package:tavern/widgets/package_tile.dart';
+import 'package:tavern/widgets/package_list_view.dart';
 import 'package:tavern/widgets/platform_filter.dart';
-import 'package:tavern/widgets/pub_logo.dart';
 import 'package:tavern/widgets/search_bar.dart';
+import 'package:tavern/widgets/tavern_logo.dart';
 
 class Home extends StatefulWidget {
   @override
   _HomeState createState() => _HomeState();
 }
 
 class _HomeState extends State<Home> with TickerProviderStateMixin {
-  final _htmlParsingClient = PubHtmlParsingClient();
-
-  Page firstPage;
-
-  /// Defaults to 1 for the first page
-  int currentPage = 1;
-
-  List<FullPackage> packagesFromPage = [];
+  HomeBloc homeBloc;
```

Comment:
homeBloc should probably be private as well

Reply:
Done already. This is on an old commit